Typo and documentation corrections

pinam45 opened the pull request:
Typo and documentation corrections

pinam45 committed:
Typo and documentation corrections

pinam45 commented:
There is also [upmethodology-fmt.sty at line 956](https://github.com/gallandarakhneorg/tex-upmethodology/blob/master/src/upmethodology-fmt.sty#L956) but I am not sure if **"this"** should be **"with"** or **"in"**.

gallandarakhneorg commented:
Thank you for your contributions.

Before beeing merged, please update your pull request according to the explanations in the `CONTRIBUTING.textile` file.
You should update the `AUTHORS` and `README.textile` files by adding your name inside.
Thx.

gallandarakhneorg commented:
Regarding the line 956, I propose: "Build a date that is following the format supported by tex-upmethodology".

pinam45 committed:
Documentation corrections

pinam45 committed:
Package documentation corrections

pinam45 committed:
Contributor addition

pinam45 commented:
I added some corrections to the code and package documentation, and updated the pull request according to the `CONTRIBUTING.textile` file (added my name in alphabetical order).

I also changed the 9 non-breaking spaces (unicode U+00A0) of the package documentation into normal spaces.

gallandarakhneorg commented on doc/upmethodology-doc.tex:
```diff
@@ -1130,18 +1130,18 @@ \section{Text formatting}
 \begin{itemize}
 \item \texttt{{\textbackslash}textsup\{text\}} \\
-	put a text as exponent in text mode instead of the basic \LaTeX exponent in math mode. In opposite to the standard \LaTeX\ macro \texttt{{\textbackslash}textsuperscript}, this macro adds an extra space after the macro when needed. \\
+	put a text as exponent in text mode instead of the basic \LaTeX\ exponent in math mode. In opposite to the standard \LaTeX\ macro \texttt{{\textbackslash}textsuperscript}, this macro adds an extra space after the macro when needed. \\
```
The `\LaTeX`macro should be followed by a space macro. Otherwise no space will be rendered after the macro. For example, `\LaTeX abc` is rendered to `LaTeXabc` in the PDF file. Please revert your changes for all the occurrences of `\LaTeX`.

gallandarakhneorg reviewed (changes requested):
Each change is valid.
